### PR TITLE
Website using flexbox

### PR DESCRIPTION
**Pull request by uursina**
I had difficulties when using the flexbox to distinguish the parent from the child element. It finally worked out however, the images are still not adapting their size proportionally when switching from one screen size to the other - from mobile phone size to tablet and then desktop size. Hope to get some good advice and feedback! Thanks in advance :)

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -8,11 +8,63 @@
   </head>
```
Comment (HIPPIEKICK) [inline code/index.html:8]: Don't forget the meta tag ;) Also - change the title to something that fits your site
`<meta name="viewport" content="width=device-width, initial-scale=1">`

File: code/index.html [hunk 2]
```diff
@@ -8,11 +8,63 @@
...
+          <img src="kitchen.jpg" alt="kitchen">
+          <h2>KITCHEN</h2>
+        </a>
+      </section>
     </main>
+
+    <section class="more">
+      <a href="#">
+      <h3>MORE CATEGORIES</h3>
+      </a>
```
Comment (HIPPIEKICK) [inline code/index.html:56]: Remember to indent one level for every descendant:
```
<a href="#">
  <h3>MORE CATEGORIES</h3>
</a>
```

File: code/style.css [hunk 3]
```diff
@@ -1,5 +1,209 @@
...
+
+/*mobile*/
+@media (max-width:667px) {
+  footer {
+    flex-direction: column;
+    height: 200px;
+    margin:0px;
+    padding: 40px;
+  }
+}
```
Comment (HIPPIEKICK) [inline code/style.css:209]: If you write your code mobile first you would only need two media queries (one for tablet, one for desktop - and the mobile would be default). Read more here: https://medium.com/@Vincentxia77/what-is-mobile-first-design-why-its-important-how-to-make-it-7d3cf2e29d00

File: code/style.css [hunk 4]
```diff
@@ -1,5 +1,209 @@
...
+  }
+
+  .container a {
+    width: 30%;
+    flex-grow: 1;
+  }
+
+  .container img {
+    max-height: 80%;
+    width: auto;
```
Comment (HIPPIEKICK) [inline code/style.css:174]: If you remove `width: auto;` your images will only take up the space that they're allowed

File: code/style.css [hunk 5]
```diff
@@ -1,5 +1,209 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+@import url('https://fonts.googleapis.com/css2?family=Cormorant:wght@300&family=Parisienne&display=swap');
 
 body {
-  background-color: peachpuff;
+  background-color:#edd8c5;
+  background-image: linear-gradient(90deg, transparent 50%, rgba(255,255,255,.5) 50%);
```
Comment (HIPPIEKICK) [inline code/style.css:5]: Super cool background! Impressive ⭐

**Review by HIPPIEKICK — COMMENTED** [review]
Good job! I think you managed well with the flexbox. I wrote a comment on how you could make the images fit better - hope that helps. 

Keep up the good work and continue exploring all the awesome things you can do with flexbox. Here are two games you could play to practice, they are actually quite fun 🤩 
- [Flexbox Froggy](https://flexboxfroggy.com/) 
- [Flexbox Zombies](https://flexboxzombies.com/p/flexbox-zombies)

**Comment by uursina** [comment]
Thank you so much Matilda for your feedback!! ⭐

**Pull request closed without merging**